lib/install/management/resource_pool.go

```diff
@@ -93,6 +93,57 @@ func setResources(spec *types.ResourceAllocationInfo, resource types.ResourceAll
 	}
 }
 
+func (d *Dispatcher) destroyVMGroup(conf *config.VirtualContainerHostConfigSpec) error {
+	defer trace.End(trace.Begin("", d.op))
+
+	if !conf.UseVMGroup {
+		return nil
+	}
+
+	d.op.Debugf("Checking for existence of DRS VM Group %s on %s", conf.VMGroupName, d.session.Cluster)
+
+	var clusterConfig mo.ClusterComputeResource
+	err := d.session.Cluster.Properties(d.op, d.session.Cluster.Reference(), []string{"configurationEx"}, &clusterConfig)
+	if err != nil {
+		d.op.Warnf("Unable to obtain cluster config: %s", err)
+		return nil
+	}
+
+	groupExists := false
+	clusterConfigEx := clusterConfig.ConfigurationEx.(*types.ClusterConfigInfoEx)
+	for i := range clusterConfigEx.Group {
```

Comment:
Is there any particular reason you use pattern like this? It is technically correct, but makes code larger and harder to read.

I would go with
```Go
for _, v := range clusterConfigEx.Group {
    if v.GetClusterGroupInfo() == conf.VMGroupName {
        groupExists = true
	break
    }
}
```